src/health_misinfo_shared/fine_tuning.py

```diff
@@ -432,7 +433,7 @@ def construct_in_context_examples(
 
     for _id, eg in _training_data.head(split_position).iterrows():
         examples += f"Input: {eg['input_text']}\n"
-        target = eg["output_text"]
+        target = json.dumps(eg["output_text"])
```

Comment:
I think this will make the summary code below break. It'd need to be converted to a dict after adding the summary. We do probably want to get rid of the summary though...

Reply:
Oh sorry! You’re absolutely right